modules/lib/lib-project/src/main/resources/lib/xp/examples/project/delete.js

```diff
@@ -0,0 +1,28 @@
+// BEGIN
+// Deletes an existing content project
+var projectLib = require('/lib/xp/project');
+var result = projectLib.delete({
+    id: 'my-project'
+});
+// END
+
+// BEGIN
+// Deletes an existing content project inside context with `system.admin` role
+var projectLib = require('/lib/xp/project');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['projectLib' is already defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5502328)